### ISSUE TITLE
LOOT incorrectly prompts to install Lock Overhaul Ordinator version

### ISSUE THREAD

**Issue by zarantha** [loot/skyrimse#1635]
Skyrim SE

I have installed the Ordinator version of Lock Overhaul, but LOOT still tells me to use the Ordinator patch from Lock Overhaul's download page. The ESPs are named the same. I tried downloading the non Ordinator version and get the same loot message. I suspect CRCs have changed?

No Ordinator version:
Lock_Overhaul.esp  B7986707
You seem to be using Ordinator - Perks of Skyrim, but you have not enabled a compatibility patch for this mod. A compatibility patch is provided on this plugin's mod page.

Ordinator version:
Lock_Overhaul.esp  60784B4A
You seem to be using Ordinator - Perks of Skyrim, but you have not enabled a compatibility patch for this mod. A compatibility patch is provided on this plugin's mod page.

**Comment by sibir-ine** [comment]
My bad. When I added the message a while back, I did it in a way that didn't take false positives into account. The Ordinator version hasn't been updated since I added the message; your plugin must've gotten edited at some point &, since the message's condition checks if the checksum of the plugin isn't that of the unedited Ordinator version, it'll give you a false positive like you received. I'll change it to check if the checksum is that of the non-Ordinator version instead.
